src/PendingScopedFeatureInteraction.php

```diff
@@ -146,7 +165,7 @@ public function allAreActive($features)
 
         return Collection::make($features)
             ->crossJoin($this->scope())
-            ->every(fn ($bits) => $this->driver->get(...$bits) !== false);
+            ->every(fn ($bits) => ($result = $this->driver->get(...$bits)) !== false && ! $result instanceof FeatureDoesNotMatchScope);
```

Comment:
This shouldn't be here, unless we add it to all of the methods. Going to put this back into draft to see if I can figure out how to make this work via `Driver@get()` instead